ring/src/lib.rs

```diff
@@ -35,21 +38,36 @@ pub fn find_complement_point<Curve: SWCurveConfig>() -> Affine<Curve> {
     }
 }
 
+pub fn find_random_point<F: PrimeField, P: AffineRepr<BaseField = F>>() -> P {
+    let mut x: u8 = 0;
+    loop {
+        let p = P::from_random_bytes(&[x]);
+        if p.is_some() && !p.unwrap().is_zero() {
+            // && !p.unwrap().is_in_correct_subgroup_assuming_on_curve() {
```

Comment:
Also, if this is used as the `seed` for SW form, this should be a random point **not** in the prime order subgroup. (we don't want sum to end up in point infinity, which is not well defined).

Reply:
Alright. But what about production? We need a point to put in the specs.
I guess that given what you just explained we can choose any random point in the prime subgroup as the seed?

Makes sense to keep `find_complement_point` function? Looks like is not used anymore if we switch to your `find_random_point` function

Reply:
@drskalman ?

Reply:
> we can choose any random point in the prime subgroup as the seed?

As long as we make sure nobody knows the discrete logarithm of it. I guess it can come from a respectable hash to curve function. 

about `find_complement_point` I'm not sure, I tried to make the test generic so I used `find_random_point` but maybe someone interested in SW would like to take advantage of that.

Reply:
Well my intent is to produce it with a simple try and increment and make the source available to the public for easy reproduction. Much like what I've done for the padding point here:

https://github.com/davxy/bandersnatch-vrfs-spec/blob/main/assets/padding/src/main.rs